packages/cli/src/commands/project/list.ts

```diff
@@ -48,39 +51,69 @@ export default async function list(
   const elapsed = ms(Date.now() - start);
 
   output.log(
-    `${list.length > 0 ? 'Projects' : 'No projects'} found under ${chalk.bold(
-      contextName
-    )} ${chalk.gray(`[${elapsed}]`)}`
+    `${
+      projectList.length > 0 ? 'Projects' : 'No projects'
+    } found under ${chalk.bold(contextName)} ${chalk.gray(`[${elapsed}]`)}`
   );
 
-  if (list.length > 0) {
-    const cur = Date.now();
-    const header = [['', 'name', 'updated'].map(title => chalk.dim(title))];
-
-    const out = table(
-      header.concat(
-        list.map(secret => [
-          '',
-          chalk.bold(secret.name),
-          chalk.gray(`${ms(cur - secret.updatedAt)} ago`),
-        ])
-      ),
+  if (projectList.length > 0) {
+    const tablePrint = table(
+      [
+        [
+          'Project Name',
+          inspect ? 'Inspect Project' : 'Latest Production URL',
+          'Updated',
+        ].map(header => chalk.bold(chalk.cyan(header))),
+        ...projectList
+          .map(project => [
+            [
+              terminalLink.isSupported
+                ? terminalLink(
+                    chalk.bold(project.name),
+                    getInspectUrl(contextName, project.name)
+                  )
+                : chalk.bold(project.name),
+              inspect
+                ? getInspectUrl(contextName, project.name)
+                : getLatestProdUrl(project),
+              chalk.gray(ms(Date.now() - project.updatedAt)),
+            ],
+          ])
+          .flat(),
+      ],
```

Comment:
**suggestion (non-blocking):** break up complex expressions

This is getting to be a pretty complex set of expressions, which is hard to follow. I'd be happy to pair on refactoring this to make it easier to follow.

Reply:
Now that the `terminal-link` logic is removed, do you still feel this is the case?

Reply:
Personally, I'd still break it up, but it is significantly leaner now. I'm good with this.